drivers/SmartThings/zwave-thermostat/profiles/aeotec-radiator-thermostat.yml

```diff
@@ -14,22 +14,11 @@ components:
       values:
         - key: "heatingSetpoint.value"
           range: [8, 28]
-  - id: thermostatCoolingSetpoint
```

Comment:
I removed these capabilities because I think they were added by accident at some point. This device doesn't have a cooling set point, humidity, etc.

These capabilities now match the base-radiator-thermostat profile, but the embedded configs are different